src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -14,11 +14,60 @@ public class Builder : IDisposable
 		public bool IsUnix { get; set; }
 		public bool RunningMSBuild { get; set; }
 		public LoggerVerbosity Verbosity { get; set; }
-		public string LastBuildOutput { get; set; }
 		public TimeSpan LastBuildTime { get; protected set; }
 		public string BuildLogFile { get; set; }
 		public bool ThrowOnBuildFailure { get; set; }
 
+		//TODO: remove
+		public string LastBuildOutput {
+			get { throw new NotImplementedException (); }
```

Comment:
This PR leaves a couple methods that would be removed if we moved forward.

I left them here with `NotImplementedException` so it would compile.